components/forms/ReviewForm.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable no-console */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import PaypalButton from './PaypalButton';
+
+class ReviewForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoaded: false,
+      params: {
+        env: '',
+        client: {
+          sandbox: '',
+          production: '',
+        },
+      },
+    };
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  async fetchData() {
+    // Fetch auth information to proceed with PayPal sucessfully
+    // We will need to manually update the url once the function exists in production
+    const url = 'https://finalize-donations-page--texasjusticeinitiative.netlify.com/.netlify/functions/paypal_params';
+    const res = await fetch(url);
+    const params = await res.json();
+    this.setState({
+      isLoaded: true,
+      params,
+    });
+  }
+
+  render() {
+    const { formState, total, returnToForm } = this.props;
+    const { state } = this;
+    const { params, isLoaded } = state;
+    const { env, client } = params;
+
+    if (!isLoaded) {
+      return (
+        <DonationReview>
+          <header>
+            <h2>Review your Information</h2>
+          </header>
+          <p>Loading...</p>
+        </DonationReview>
+      );
+    }
+
+    const onSuccess = payment => {
```

Comment:
Do we need these `onSuccess`, `onCancel`, and `onError` functions? The console logs seem like they'd be useful for our testing, but I'm not sure they add anything for us in production.

Reply:
Agreed. This may be something to revisit in the future in order to provide some feedback to the user if there is an error, but I think removing them is alright for now.